MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include CHANGELOG.rst
+include LICENSE.txt
+include README.rst
+include requirements/*
+include apparmor-profiles/*
+include sudoers-file/*
+recursive-include {{cookiecutter.sub_dir_name}} *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg
```

Comment:
cookiecutter here seems wrong, and I don't think any of these file types are in this repo.

Reply:
Whoops, this was just bad copy-pasting on my part:

> recursive-include {{cookiecutter.sub_dir_name}} ...

the cookiecutter would've templated that to:

> recursive-include codejail ...

Anyway, I'll just remove that line since it's not relevant to us.